src/Representation/Operands/ConstantFloat.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Symbolica.Abstraction;
+using Symbolica.Expression;
+
+namespace Symbolica.Representation.Operands
+{
+    public sealed class ConstantFloat : IOperand
+    {
+        private readonly Bits _size;
+        private readonly string _value;
+
+        public ConstantFloat(Bits size, string value)
```

Comment:
This will be needed when we switch the deserialization over to the `LLVMSharp` bindings (in the next PR). When using the C++ bindings directly we could use `bitcastToAPInt` on `APFloat` (Arbitrary Precision) to get the bit pattern, and then we could just handle them the same as integer constants. However, the C# bindings do not seem to expose this, so we have to be able to parse any of the float [notations](https://llvm.org/docs/LangRef.html#simple-constants) it could throw at us.